app/components/candidate_interface/course_choices_review_component.html.erb

```diff
@@ -45,6 +45,22 @@
           to see if the course will re-open or discuss alternatives
         </li>
       </ul>
+    <% elsif course_choice.chosen_study_mode_full? %>
+      <p class='app-review-warning__primary-message'><%= course_choice.chosen_study_mode_full_error %>.</p>
```

Comment:
This might be a good opportunity to rename `course_choices` -> `application_choices`, likely we've started doing elsewhere.